transmit-lib/src/cmr/transmit/urs.clj

```diff
@@ -69,12 +70,15 @@
   (let [{:keys [status body]} (request-with-auth context {:url-fn #(user-info-url % user)
                                                           :method :get
                                                           :raw? true})]
-   (when-not (= 200 status)
-     (errors/internal-error!
-      (format "Cannot get info for username [%s] in URS. Failed with status code [%d].
-               EDL error message: [%s]"
-              user status (pr-str body))))
-   body))
+    (when-not (= 200 status)
+      (info (format "Cannot get info for username [%s] in URS. Failed with status code [%d].
+        EDL error message: [%s]" user status (pr-str body)))
```

Comment:
what's in (pr-str body), there is nothing like a token right?

Reply:
Related to above answer - the response is a big HTML block, `There has been an error processing your request. ...` So no, no user or request info.